python/lsst/obs/subaru/strayLight/rotatorAngle.py

```diff
@@ -61,8 +61,8 @@ def _mjd2gmst(mjd):
     """convert mjd to GMST(Greenwich mean sidereal time)"""
     mjd_f = mjd % 1
     jc2000 = _mjd2jc2000(mjd)
-    gmst_s = ((6.0*3600.0 + 41.0*60.0 + 50.54841) +
-              8640184.812866*jc2000 + 0.093104*jc2000**2.0 - 0.0000062*jc2000**3.0 +
+    gmst_s = ((6.0*3600.0 + 41.0*60.0 + 50.54841) + # noqa
```

Comment:
I think you are now allowed to add an exception for 504 to your setup.cfg file.  See e.g. https://github.com/lsst/ap_verify/pull/50.